docs/apim/4.0/guides/gravitee-expression-language.md

```diff
@@ -193,6 +193,17 @@ Request context attributes and examples are listed below.
 {% endtab %}
 {% endtabs %}
 
+#### Important Note for v4 Emulation Engine Users
+When using the v4 emulation engine, it is important to note that the api-key attribute is no longer accessible in the context if you wish to use it as a logging filter condition. Instead of referencing the api-key attribute directly with expressions like `{#context.attributes['api-key'] == 'api-key-value'}`, users can retrieve the API key from request parameters or headers.
```

Comment:
Some changes:
When you use the v4 emulation engine, you retrieve the API key from the request parameters or headers instead of referencing the API directly with expressions like `{#context.attributes['api-key'] == 'api-key-value'}`.

Reply:
Can we keep my explanation please. It is really clearer I think from a user point of view

Reply:
It has to be rephrased to tell the user what the user can do rather than what the user can't do. There is no point in telling the user what they can't do since the goal is to tell the user how to do something. 

What about this note: 

When you use the v4 emulation engine, you retrieve the API Key from the request parameters or headers. You cannot reference the API directly with expressions like {#context.attributes['api-key'] == 'api-key-value'} because the api-key attribute is not accessible in the context as a logging filter expression.

Reply:
Tbh I'm still not convinced but I'm not a native english speaker so I believe in you 😅
Done ✅

Reply:
If I'm honest, that makes your opinion more valuable because our documentation needs to be accessible to non-native English speakers. 

If you want, we can hop on a quick call sometime to discuss it?